core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletDefinition.scala

```diff
@@ -46,23 +46,16 @@ case class StreamletDefinition(appId: String,
 }
 
 object Topic {
-  //TODO  used for testing, remove
-  def apply(appId: String, streamletRef: String, outlet: String): Topic =
-    Topic(appId, streamletRef, s"$appId.$streamletRef.$outlet", ConfigFactory.empty(), None)
+  def apply(name: String): Topic = Topic(name, ConfigFactory.parseString("managed=true"))
```

Comment:
These default values spread here-and-there make me a bit nervous. 
We might want to refactor this (later) to bring these defaults together.